src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -119,6 +125,31 @@ static MvcCoreLoggerExtensions()
                 2,
                 "Executed action method {ActionName}, returned result {ActionResult}.");
 
+            _executingFilters = LoggerMessage.Define<string, string[]>(
```

Comment:
Ok I think I know part of why I was confused earlier. This logging helper is called "Executing Filters" (the field name, the message, the extension method, etc.), but it's not executing anything at all. It's just the "execution plan."

Maybe these could just be renamed to something that doesn't make it sound like it's actually running anything? Like `_logFilterExecutionPlan` etc.? (There are many possibilities on what to name this.)